### PR TITLE
DOC-920 Connect to prometheus operator

### PR DESCRIPTION
**Pull request by kaitlynmichael**
Article on how to configure Prometheus operator to target the "prometheus" port. Includes content about new service created for the Jolly Roger release (RED-61807).

Staging site: https://docs.redis.com/staging/jira-doc-920/kubernetes/re-clusters/connect-prometheus-operator/
Jira tickets: DOC-920 and DOC-1112

### CODE REVIEW

**Comment by laurentdroin** [comment]
I can't access the staging site.

**Comment by kaitlynmichael** [comment]
> I can't access the staging site.

So sorry! I just fixed it. This link should work now: https://docs.redis.com/staging/jira-doc-920/kubernetes/re-clusters/connect-prometheus-operator/

**Review by bascar — COMMENTED** [review]
Typo missing an "r", Line 26,  (`<rec_name>-pom`) 
should be:  (`<rec_name>-prom`)

**Comment by laurentdroin** [comment]
"To collect and display metrics data from your databases".

- I would just use "collect". Prometheus has a very basic UI but the displaying of the data is best left to Grafana.
- With Prometheus, you collect metrics not just for your databases, but for your Redis Enterprise Cluster as well.

File: content/kubernetes/re-clusters/connect-prometheus-operator.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+To collect  metrics data from your databases and Redis Enterprise cluster (REC), you can connect your [Prometheus](https://prometheus.io/) server to an endpoint exposed on your REC. The Redis Enterprise operator creates a dedicated service to expose the `prometheus` port (8070) for data collection. A custom resource called `ServiceMonitor` allows the [Prometheus operator](https://github.com/prometheus-operator/prometheus-operator/tree/main/Documentation) to connect to this port and collect data from Redis Enterprise.
+
+## Prerequisites
+
+Before connecting Redis Enterprise to Prometheus on your Kubernetes cluster, make sure you've done the following:
+
+- [Deploy the Redis Enterprise operator]({{<relref "/kubernetes/deployment/quick-start.md">}}) (version 6.2.10-tbd or newer)
+- [Deploy the Prometheus operator](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md) (version 0.19.0 or newer)
+- [Create a Redis Enterprise cluster]({{<relref "/kubernetes/deployment/quick-start.md#create-a-redis-enterprise-cluster-rec">}})
```
Comment (rrelledge) [inline content/kubernetes/re-clusters/connect-prometheus-operator.md:22]: The anchor link wasn't working as expected.
```suggestion
- [Create a Redis Enterprise cluster]({{<relref "/kubernetes/deployment/quick-start#create-a-redis-enterprise-cluster-rec">}})
```

File: content/kubernetes/re-clusters/connect-prometheus-operator.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+## More info
+
+- github.com/prometheus-operator
+  - [Getting started](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md)
+  - [Running exporters](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/running-exporters.md)
+  - [Related resources](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md#related-resources)
+  - [Troubleshooting ServiceMonitor changes](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/custom-metrics-elements.png)
+- docs.redis.com
+  - [Metrics in Prometheus]({{<relref "/rs/administering/monitoring-metrics/prometheus-metrics-definitions.md">}})
+  - [Monitoring and Metrics]({{<relref "/rs/administering/monitoring-metrics/_index.md">}})
```
Comment (rrelledge) [inline content/kubernetes/re-clusters/connect-prometheus-operator.md:71]:
```suggestion
  - [Monitoring and metrics]({{<relref "/rs/administering/monitoring-metrics/_index.md">}})
```

**Review by rrelledge — APPROVED** [review]
Looks good to me. I just left a few small suggestions.

**Pull request merged**